microcloud/cmd/microcloud/ask.go

```diff
@@ -702,9 +719,18 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 						return err
 					}
 
-					config[gateway] = fmt.Sprintf("%s-%s", rangeStart, rangeEnd)
+					ipConfig[gateway] = fmt.Sprintf("%s-%s", rangeStart, rangeEnd)
+
+					dnsAddresses, err := c.asker.AskString("Specify the default DNS addresses (comma-separated IPv4 addresses) for this OVN network instead of using the default gateway address? (empty to skip): ", "", commaValidator)
```

Comment:
Why only IPv4?

Reply:
This option is specifying the IPs of DNS servers reachable on the uplink network that will be used for external DNS resolution by OVN networks using that uplink.

So how about:

> Specify the DNS servers (comma-separated IP addresses) for OVN networks to use instead of using the default gateway address? (empty to skip):

Also, have we mentioned OVN in `microcloud init` up until this point before?

Reply:
Good catch thanks. I'll add IPv6.